.github/workflows/pr-update-on-author-input.yml

```diff
@@ -0,0 +1,14 @@
+name: Update PR on author input
+
+on:
+  - issue_comment
+  - pull_request_review_comment
```

Comment:
Don't they double each other? As in: isn't `pull_request_review_comment` also an `issue_comment`?

Reply:
nope, `pull_request_review_comment` would be triggered on comments to the diff.